### PR TITLE
net-vpn/i2pd: version bump 2.47.0 and 2.46.1 stabilization amd64

### PR DESCRIPTION
**Pull request by khumarahn**
Hi!

No changes here. 2.46.1 has been running well, no major bugs I'm aware of

### CODE REVIEW

File: net-vpn/i2pd/i2pd-2.46.1.ebuild [hunk 1]
```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/PurpleI2P/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+KEYWORDS="amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
```
Comment (juippis) [inline net-vpn/i2pd/i2pd-2.46.1.ebuild:14]: Hey, we don't let users stabilize packages as it follows a "strict" process we can't be sure you've followed. Please file a stabilization bug at our bugzilla, I can help do that if needed.

Comment (khumarahn) [reply net-vpn/i2pd/i2pd-2.46.1.ebuild]: ah, sorry. Removed the stabilization commit, will create a bug

**Review by juippis — APPROVED** [review]
Thanks!

**Comment by thesamesam** [comment]
juippis merged the bump already but I'm merging the cleanup now too. Thanks!

**Pull request closed without merging**